Services/CO.CDP.Organisation.WebApi/Model/Exceptions.cs

```diff
@@ -9,4 +9,6 @@ public class InvalidUpdateSupplierInformationCommand(string message, Exception?
 public class InvalidQueryException(string message, Exception? cause = null) : Exception(message, cause);
 public class InvalidUpdateConnectedEntityCommand(string message, Exception? cause = null) : Exception(message, cause);
 
-public class UnknownConnectedEntityException(string message, Exception? cause = null) : Exception(message, cause);
\ No newline at end of file
+public class UnknownConnectedEntityException(string message, Exception? cause = null) : Exception(message, cause);
+
+public class MissingOrganisationIdException(string message, Exception? cause = null) : Exception(message, cause);
```

Comment:
This should result in a 404, so needs to be mapped to an error code. Like here https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/3a84adb9f963de84cc8a51ffcb5b4e9f2e2f6593/Services/CO.CDP.Organisation.WebApi/Extensions/ServiceCollectionExtensions.cs#L11

Reply:
done.